about.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Global Summit About Page</title>
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div class="mobile-navbar">
+            <nav class="mobile-navbar-container">
+                <span class="hamburger-icon"><img src="images/hamburger.png" alt="hamburger icon"></span>
+                <img src="images/Icon.png" class="close-icon-container" alt="close icon">
+                <ul class="nav-links">
+                    <li><a href="/index.html">Home</a></li>
+                    <li><a href="#Partner">Partner</a></li>
+                    <li><a href="#">Join</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                    <li><a href="#">News</a></li>
+                    <li><a href="#">CC Campaign</a></li>
+                </ul>
+            </nav>
+            <a href="#"><img src="images/header-logo.png" alt="Global Summit Logo"></a>
+        </div>
+        <ul class="navbar-banner">
+            <li><a href="#"><img src="images/facebook-logo.png" class="mini-navbar-img" alt="facebook logo"></a></li>
+            <li><a href="https://twitter.com/AyokunnumiA"><img src="images/twitter-logo.png" class="mini-navbar-img"
+                        alt="twitter logo"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Logo</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+
+        <nav class="desktop-navbar">
+            <div><a href="/index.html"><img src="images/header-logo.png" alt=""></a></div>
+            <ul class="desktop-navbar-list">
+                <li><a href="/about.html" class="desktop-navbar-link">About</a></li>
+                <li><a href="#" class="desktop-navbar-link">Program</a></li>
+                <li><a href="#" class="desktop-navbar-link">Join</a></li>
+                <li><a href="#" class="desktop-navbar-link">Sponsor</a></li>
+                <li><a href="#" class="desktop-navbar-link">News</a></li>
+                <li><a href="#" class="desktop-navbar-link CC-Campaign-link">CC Campaign</a></li>
+            </ul>
+        </nav>
+        <div class="abt-header-section">
+            <div class="abt-theme">
+                <div class="abt-theme-content">
+                    <p class="header-theme abt-header-theme">"You Never Fail Until You Stop Trying"</p>
```

Comment:
- [ ] Please kindly center the text on the about page for mobile screens, you can see the red arrows showing a guide for  both your design and the template design   
- project design
![ct](https://github.com/Ayokunnumi1/Conference-Page/assets/89055502/d8b31aea-ac27-4b74-9595-ce11b521cb42)
- your design
![cts](https://github.com/Ayokunnumi1/Conference-Page/assets/89055502/bdbea99d-19d0-4d34-80bb-990204270813)